src/libaktualizr/primary/sotauptaneclient.h

```diff
@@ -171,6 +173,7 @@ class SotaUptaneClient {
   // ecu_serial => secondary*
   std::map<Uptane::EcuSerial, std::shared_ptr<Uptane::SecondaryInterface>> secondaries;
   std::mutex download_mutex;
+  mutable Uptane::EcuSerial primary_ecu_serial_;
```

Comment:
Why does this need to be `mutable`?

Reply:
does not have to be such anymore, leftover of one the previous versions.

Reply:
Can you still remove the `mutable` modifier, or did it turn out it was necessary?

Reply:
sure, just missed it in the first time.